cmake-tool/helpers/simulation.cmake

```diff
@@ -152,6 +152,16 @@ function(GenerateSimulateScript)
         set(sim_graphic_opt "-nographic")
         set(sim_cpu "${KernelArmCPU}")
         SetDefaultMemSize("${QEMU_MEMORY}")
+    elseif(KernelPlatformQEMURiscVVirt)
+        set(QemuBinaryMachine "qemu-system-${KernelSel4Arch}")
+        set(sim_machine "virt")
+        # defaults for the virt machine:
+        #     aclint=off (on/off)
+        #     aia=none (none/aplic/aplic-imsic)
+        #     aia-guests=0 (VS-level AIA IMSIC pages per hart)
+        set(sim_graphic_opt "-nographic")
+        set(sim_cpu "rv${KernelWordSize}")
```

Comment:
```suggestion
        set(sim_cpu "rv${KernelWordSize}")
        set(qemu_sim_extra_args "-bios none")
```

Reply:
(see Ivan's comment in https://github.com/seL4/seL4/pull/818)

Reply:
I think the better solution would be that we make the script support `-bios <image>` instead of `-kernel <image>` on RISC-V.

Reply:
Since the `-bios none` is used for the other targets also, I think it's easier to use this for now and merge things to have qemu-riscv-virt working. I will make another PR for `-bios <image>` then.

Reply:
:+1: